modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/FieldBase.scss

```diff
@@ -2,6 +2,6 @@
 
 .lfr-objects__field-base {
 	&-tooltip-icon {
-		color: #6b6c7e;
+		color: $gray-600;
```

Comment:
Hi @danvb112, please squash this change and the ones about the RadioField to the previous commits. This helps during the review and also if we need to revert commits in the future. Thanks!

Reply:
Good catch, Done! Thanks @naraandrade